### ISSUE TITLE
Critical resources cannot be fetched from CDN

### ISSUE THREAD

**Issue by eismail-aiopsgroup** [z1haze/sfcc_dev_console#7]
Very often critical resources like `https://unpkg.com/monaco-editor@0.50.0/min/vs/loader.js` cannot be fetched (502 Bad Gateway) and because of that, the console fails to load. I'm not sure if the cartridge is requesting this file, or it's loaded by an NPM package but perhaps it's better to have them stored locally if possible.

The page tries to load for a minute:
![image](https://github.com/user-attachments/assets/aef89477-08a6-47e4-a4a9-e2b86bbaacde)
![image](https://github.com/user-attachments/assets/7346a2a3-02e6-4ccf-8f2e-0e1e76184699)

When not being able to fetch the JS file, the editor is not available:
![image](https://github.com/user-attachments/assets/def26ba9-68b1-4fc2-a91c-fab08f8dbfc2)

The same happens with files like:
- `https://unpkg.com/monaco-editor@0.50.0/min/vs/editor/editor.main.css`, too.
- `https://unpkg.com/monaco-editor@0.50.0/min/vs/base/common/worker/simpleWorker.nls.js`
- etc.

So it looks like there's something wrong with [unkp.com](https://unpkg.com).
As a workaround, I open the failed files in the browser directly, and it helps for the next load. But even in this case, I'm sometimes receiving a bad gateway error.

**Comment by z1haze** [comment]
Yes, this been an intermittent issue for years now. We use moncao from cdn to avoid the hassle that comes with setting up web workers on sfcc as well as keeping the editor itself current.

It has been a few years since I've visited this issue, so maybe it is easier to dp now. If I can get some time to experiment,  I will. Otherwise, a PR is always welcome if you're feeling up to it!
